demo/site/src/styles/global.scss

```diff
@@ -34,6 +34,7 @@
     --palette-success-light: #77d06a;
     --palette-success-dark: #488938;
     --palette-success-contrastText: #090909;
+    --palette-functional-highlight-magenta: #d817fa;
```

Comment:
variable names must not contain color names, that defeats the whole purpose of them

Reply:
then please talk to UX to find a proper name for this color